### PR TITLE
string wrap part4, ruleid

### PR DESCRIPTION
**Pull request by aryx**
test plan:
see related PR in semgrep

- [x] I ran `make setup && make` to update the generated code after editing a `.atd` file (TODO: have a CI check)
- [x] I made sure we're still backward compatible with old versions of the CLI.
      For example, the Semgrep backend need to still be able to *consume* data generated
	  by Semgrep 1.17.0.
      See https://atd.readthedocs.io/en/latest/atdgen-tutorial.html#smooth-protocol-upgrades

### CODE REVIEW

File: semgrep_output_v1.atd [hunk 1]
```diff
@@ -106,7 +104,8 @@ type position
 }
 
 (* File path. less: could convert directly to Path class of pathlib library for Python *)
-type fpath <ocaml attr="deriving show"> = string wrap <ocaml module="ATDStringWrap.Fpath">
+type fpath <ocaml attr="deriving show"> =
```
Comment (emjin) [inline semgrep_output_v1.atd:107]: Is this change intentional? Looks like just whitespace

**Review by emjin — APPROVED** [review]

File: semgrep_output_v1.atd (hunk 1 above)
Comment (aryx) [reply semgrep_output_v1.atd:107]: yes just whitespace.
